Phaethon/Tests/IntegrationTests/LoginTest.cs

```diff
@@ -165,15 +145,14 @@ public async Task Edit_User_OK()
         {
             // Setup
             InitializeData();
-            name = userModel2.Username;
-            userModel2.Username = "Maybe";
-            name2 = userModel2.Username;
+            _name = _userModel2.Username;
+            _userModel2.Username = "Maybe";
+            _name2 = _userModel2.Username;
             //Act
-            var response = await _client.PostAsJsonAsync("Login/CreateOrUpdate", userModel2);
+            var response = await _client.PostAsJsonAsync("Login/CreateOrUpdate", _userModel2);
             //Assert
             Assert.AreEqual(HttpStatusCode.OK, response.StatusCode);
-            Assert.AreNotEqual(name, name2);
-            Cleanup();
+            Assert.AreNotEqual(_name, _name2);
         }
```

Comment:
Your UI pages should be using the LanguagePack, as oposed to hardcoding in text. Have a look at any other cshtml file for reference on how to implement that, or just ask me/ralfs.